### PR TITLE
Remove obsolete parts and edit desktop

### PR DESCRIPTION
**Pull request by goldim**
- removed obsolete parts (e.g. for qooxdoo 5)
- removed spaces from sidebar page
- article tech_website_apies moved to desktop
- added two links in sidebar (inline and tech_website_apies)
- edited desktop (es6, codes marked accordingly json5 or javascript, noticed and fixed some bad links and so on)

Don't know how to properly fix this sentence with error: "This renderer is more a of demo showing how easy it can be to implement", please help.

### CODE REVIEW

**Review by cboulanger — APPROVED** [review]
Again: thank you for this clean-up, very important work!

**Review by johnspackman — APPROVED** [review]

File: docs/desktop/gui/table_styling.md [hunk 1]
```diff
@@ -323,7 +323,7 @@ create your very own appearance by putting together the necessary CSS styles.
 Since the latter solution is a more complex one, I'll only explain the first
 solution which helps you in styling the table rows and columns in a quick way.
 
-```javascript
+```json5
 // these two lines have to inserted in your application color theme
```
Comment (derrell) [inline docs/desktop/gui/table_styling.md:327]: these two lines have to BE inserted

**Review by derrell — CHANGES_REQUESTED** [review]
Sorry, this requested change wasn't one from your PR but rather, adjacent to it that I noticed while reviewing.

File: docs/desktop/gui/table_styling.md (hunk 1 above)
Comment (goldim) [reply docs/desktop/gui/table_styling.md:327]: no problem, will fix it

**Review by derrell — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Pull request merged**